SuperCalculator/src/UnitTests/CalculatorTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import supercalculator.Tan;
+
+/**
+ *
+ * @author Haitham Abdel-Salam
+ */
+class CalculatorTest {
```

Comment:
Instead of calling this Unit test CalculatorTest, wouldn't be better to call it the specific_function_unit_test? In this way, other team members can work in parallel in different function without having possibly merge conflicts later on.